### ISSUE TITLE
entity iframes no longer have border next to them

### ISSUE THREAD

**Issue by dustingetz** [hyperfiddle/hyperfiddle#783]
![image](https://user-images.githubusercontent.com/124158/49397485-bc41df00-f709-11e8-97e4-57f31a4208d6.png)

Introduced in the change that lifted `[hyperfiddle.ui/field [] ctx hyperfiddle.ui/entity-links-iframe (assoc props :label-fn (contrib.reactive/constantly nil))]` out of result and into userland

**Comment by khardenstine** [comment]
Looks like a domain/css error
![image](https://user-images.githubusercontent.com/7741805/49398874-a20a0000-f70d-11e8-9998-b8e3eab65c78.png)

**Comment by dustingetz** [comment]
No, the issue is in data mode here: http://www2.hyperfiddle.net/:docs.index!subnav/~entity('$',17592186047946)

**Comment by khardenstine** [comment]
In data mode those custom styles from `.display-mode-user` would not apply, therefore you would still see the pink box.  In user mode, they do apply and are hiding the pink box.

**Comment by khardenstine** [comment]
`hf.ui.iframe/iframe-cmp` takes care of adding both the pink box styles and the alt-click handler to the props for the UI (xray or user).  Since there is no wrapping div inside iframe-cmp, the actual render function handles inserting the classes and on-click into the dom.

With that background knowledge, I don't understand how `hf.ui/field` and `hf.ui/result` have anything to do with this ticket.

Also with the URL provided in the original post, the css class has been included into the DOM; the only thing preventing the pink box from appearing is an !important CSS property setting the border to none.

**Comment by dustingetz** [comment]
![image](https://user-images.githubusercontent.com/124158/49895662-079d7100-fe1f-11e8-915a-677bd3e1b516.png)

Unchecking this repairs

**Comment by dustingetz** [comment]
Removing extra .ui here repairs 
![image](https://user-images.githubusercontent.com/124158/49895746-43383b00-fe1f-11e8-8074-d9a0bc289d20.png)
